src/parsers/trades/pionex/index.ts

```diff
@@ -0,0 +1,74 @@
+import { getCSVData } from '../../';
+import { EXCHANGES, IImport, IPartialTrade, ITrade } from '../../../types';
+import { createDateAsUTC, createID } from '../../utils';
+
+interface IPionex {
+    'date(UTC+0)': string;
+    amount: string;
+    price: string;
+    order_price: string;
+    side: string;
+    symbol: string;
+    state: string;
+    fee: string;
+    strategy_type: string;
+}
+
+export async function pionexParser(importDetails: IImport): Promise<ITrade[]> {
+    const data: IPionex[] = await getCSVData(importDetails.data) as IPionex[];
+    const internalFormat: ITrade[] = [];
+    for (const trade of data) {
+        const tradeToAdd: IPartialTrade = {};
+        tradeToAdd.date = createDateAsUTC(new Date(trade['date(UTC+0)'])).getTime();
```

Comment:
See my other comment on the other parser, should be able to combine this and the following line into one when defining tradeToAdd

Reply:
Yes will do

Reply:
Please check